extensions/blocks/videopress/utils.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Internal dependencies
+ */
+import { ASPECT_RATIOS } from './constants';
+
+import classnames from 'classnames';
+
+/**
```

Comment:
> Code is pull from Gutenberg embed block which handles it correctly
> WordPress/gutenberg@e4b6d70/packages/block-library/src/embed/util.js#L177

Maybe we could add a comment here mentioning this?

Do you think it may be worth asking in the Gutenberg repo if this utility could be made available to third-parties?

Reply:
Sure! I have initially tried to find a way to use it from Gutenberg without success.

I am not sure though whether we should wait or iterate?

Reply:
I think we could merge things are you did right now, and revisit later if it eventually becomes available via Gutenberg.

I think we should add a comment here referring to the original, though, this will make it easier to track things on the long run.

Reply:
Thanks, added a comment https://github.com/Automattic/jetpack/blob/ee5457f1bcbf20409b11ba11b21aa149cc18085c/extensions/blocks/videopress/utils.js#L1-L4